modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -1230,4 +1245,37 @@ public ClusterSnapshotFuture(UUID rqId, String name) {
             return super.onDone(res, err, cancel);
         }
     }
+
+    /** Start creation of cluster snapshot closure. */
+    private static class CreateSnapshotClosure implements IgniteClosure<String, Void> {
```

Comment:
Should this task be marked as `@GridInternal` ?

Reply:
Added. Thank you!